src/content/docs/synthetics/synthetic-monitoring/getting-started/new-runtime.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Convert to the new synthetic runtimes
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Runtime conversion
+metaDescription: "Synthetic monitoring's new runtime requires that you make some changes to your already existing monitors."
+---
+
+We're introducing a new runtime for public monitoring locations. We plan to apply the new runtime to private monitors by the end of 2022. While we're providing backwards support for monitors using the Legacy runtime and its features, we recommend converting to the current runtime as soon as possible. 
+
+Going forward, new monitors are defaulting to the new runtime while monitors created using the Legacy version will need additional configuration.
+
+<Callout variant="important">
+Before switching to the new runtime, confirm that you've updated your allow lists to accept network requests from the new runtime’s [public IP ranges](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips).
+</Callout>
+
+## Benefits [#benefits]
+
+Converting to the new runtime gives you: 
+
+- **Access to newer versions of Google Chrome.** With planned updates twice a year, the new runtime allows us to keep Chrome up to date so your monitors use the most recent Chrome version.   
+- **Access to the updated scripted API runtime.** This includes updates to newer versions of node.js. 
+- **Support for async / await syntax.** This simplifies script creation and maintenance, diverging from the old runtime’s `.then` promise chaining.
+- **Support for shadow DOM objects.** Updated runtime treats shadow DOM updates similar to frames, allowing your monitors to find objects inside the shadow DOM.
```

Comment:
I'd say "The new runtime..." because that's what you've used in this doc up to now

Also, why not "...treats shadow DOM like frames..."? It's always worth considering the extra words you can cut.